### PR TITLE
ci(PR): manage validator checks in `PR.yml`

### LINKED ISSUE #3765: Trigger live checks on workflow level
### File Location(s)

https://github.com/gear-tech/gear/blob/master/.github/workflows/validation.yml

### reason 1

This action is actually of reason of that the `label / dispatch` CI is easy to be broken xd, for example in this check suite,

https://github.com/gear-tech/gear/runs/22103880455 of #3764 

`build / linux` has been queued to the check suite `Live check on Vara Network Validator machine`, however, the top namespace of it has been skipped already https://github.com/gear-tech/gear/actions/runs/8088892853, so the result of `build / linux` didn't show up in the UI

### reason 2

This action is not used frequently, hide the long name

### PR DESCRIPTION
**Pull request by clearloop**
Resolves #3765 
Closes #3820 

triggers validator-checks inside of `PR.yml` in case of creating empty check suite in github UI

@gear-tech/dev

### CODE REVIEW

**Review by breathx — APPROVED** [review]
Please actualise PR according to #3616 from master

Otherwise lgtm

**Comment by breathx** [comment]
whats wrong with workflow? should be fixed?

**Comment by clearloop** [comment]
> whats wrong with workflow? should be fixed?

interesting, no logic changes in this workflow, only 5 PRs(including the current one) labeled `check-validators` in history, but the most recently one https://github.com/gear-tech/gear/pull/3801 seems working with it perfectly

File: .github/workflows/PR.yml [hunk 1]
```diff
@@ -68,3 +69,13 @@ jobs:
...
+
+  validator:
+    needs: status
+    if: ${{ needs.status.outputs.validator == 'true' }}
+    uses: ./.github/workflows/validation.yml
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      VARA_VALIDATOR_8: ${{ secrets.VARA_VALIDATOR_8 }}
+      SSH_VARA_USERNAME: ${{ secrets.SSH_VARA_USERNAME }}
+      VARA_SSH_PRIVATE_KEY: ${{ secrets.VARA_SSH_PRIVATE_KEY }}
```
Comment (clearloop) [inline .github/workflows/PR.yml:81]: https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

**Review by ark0f — APPROVED** [review]

**Pull request merged**